src/node_modules/pallete.js

```diff
@@ -1,28 +1,25 @@
 module.exports = {
```

Comment:
this file shouldn't exist (it's not a module), a better place given the content is probably `src/node_modules/ui/pallete.js`?